testsuite/cluster-test/src/health/mod.rs

```diff
@@ -57,11 +58,12 @@ impl fmt::Debug for ValidatorEvent {
     }
 }
 
+#[async_trait]
 pub trait HealthCheck {
     /// Verify specific event
     fn on_event(&mut self, event: &ValidatorEvent, ctx: &mut HealthCheckContext);
     /// Periodic verification (happens even if when no events produced)
-    fn verify(&mut self, _ctx: &mut HealthCheckContext) {}
+    async fn verify(&mut self, _ctx: &mut HealthCheckContext);
```

Comment:
I changed verify back to declaration is because of the known warning of Rust if we use default implementation:
```
warning: the trait `health::HealthCheck` cannot be made into an object
  --> testsuite/cluster-test/src/health/mod.rs:66:14
   |
62 | pub trait HealthCheck {
   |           ----------- this trait cannot be made into an object...
...
66 |     async fn verify(&mut self, _ctx: &mut HealthCheckContext) {}
   |              ^^^^^^ ...because method `verify` references the `Self` type in its `where` clause
```
Refer to:
[https://github.com/rust-lang/rust/issues/51443](url)
[https://doc.rust-lang.org/error-index.html](url) Chapter -> Method references the Self type in its parameters or return type

Reply:
ok :)